10up/insecure-content-warning
# fix/53: check content for Ctrl+S and Command+S

Closes #53 



### Description of the Change



This PR enables the checks when saving through the `primary + S` shortcut.

### Benefits

This benefits the user from not saving insecure post content to the DB using the shortcut.


### Verification Process



1. In the fix branch, add an image to the editor on an HTTPS site.
2. Edit the image as HTML and replace the protocol to HTTP.
3. Try saving the post using the **Publish** or **Update** button.
4. Observe the error notice saving **1 insecure element found.**
5. Repeat steps 1 and 2 and try saving the post using `primary + S` and repeat step 4.
6. In develop branch, `primary + S` will save the post with insecure element.

### Credits

@Sidsector9 @dinhtungdu @jeffpaul @faisal-alvi 

### Checklist:






### Changelog Entry

Fix: Issue with saving post with insecure element using the `primary + S` shortcut.
